src/lockscreen.js

```diff
@@ -201,8 +201,19 @@ var Lockscreen = GObject.registerClass({
 
         this._updateBackground();
 
-        if (!this._manager.isUnlocked(this._lock))
+        try {
+            if (!this._manager.isUnlocked(this._lock))
+                return;
+        } catch (e) {
+            logError(e, 'Error trying to load unlock state, trying again in 100 milisecond');
+            GLib.timeout_add(GLib.PRIORITY_HIGH, 100,
```

Comment:
Nitpick: Don't repeat the number in the comment/timeout (as it's easy to get them unsynced in the future). Maybe the following works?
```
const next_try_timeout = 100;
logError(e, 'Error trying to load unlock state, trying again in %s milisecond', next_try_timeout);
GLib.timeout_add(GLib.PRIORITY_HIGH, next_try_timeout, ...)
```

Reply:
Ok, let's do that.